### DISCUSSION TITLE
什么时候更新一下miraimc的正式版

### DISCUSSION THREAD

**Discussion by Natsusomekeishi** [DreamVoid/MiraiMC#339] [普通 / General]
距离上次更新已经过去2个多月了，mirai的无法登录的临时处理方案也增加了新内容，希望可以适配一下最新版，作为小白服主自行适配新版本的依赖有些难办

**Comment by DreamVoid** [comment]
给钱就更

**Reply by Natsusomekeishi** [reply]
请问是近期较忙的原因嘛，还是已经停更了？

**Reply by Natsusomekeishi** [reply]
如果需要赞助更新，需要多少钱？

**Reply by DreamVoid** [reply]
> 如果需要赞助更新，需要多少钱？

1000人民币马上发新版本

**Reply by Natsusomekeishi** [reply]
> > 如果需要赞助更新，需要多少钱？
> 
> 1000人民币马上发新版本

这…有点太贵了叭

**Reply by cssxsh** [reply]
> 给钱就更

@DreamVoid 

使用文档里似乎没有 调整 mirai-core 版本的 相关方法的介绍
这是否是一个未实现的功能?

我看源码似乎是 从 maven 下载然后 URLClassLoader 加载
是可以指定版本的

**Reply by DreamVoid** [reply]
> > 给钱就更
> 
> @DreamVoid
> 
> 使用文档里似乎没有 调整 mirai-core 版本的 相关方法的介绍 这是否是一个未实现的功能?
> 
> 我看源码似乎是 从 maven 下载然后 URLClassLoader 加载 是可以指定版本的

配置文件里的mirai-core-version就是指定版本的

**Reply by cssxsh** [reply]
> > 给钱就更
> 
> @DreamVoid
> 
> 使用文档里似乎没有 调整 mirai-core 版本的 相关方法的介绍 这是否是一个未实现的功能?
> 
> 我看源码似乎是 从 maven 下载然后 URLClassLoader 加载 是可以指定版本的

好吧找到了, 藏在配置文件的介绍里

**Reply by cssxsh** [reply]
> > > 给钱就更
> > 
> > 
> > @DreamVoid
> > 使用文档里似乎没有 调整 mirai-core 版本的 相关方法的介绍 这是否是一个未实现的功能?
> > 我看源码似乎是 从 maven 下载然后 URLClassLoader 加载 是可以指定版本的
> 
> 配置文件里的mirai-core-version就是指定版本的

有个小建议
镜像库的选择可以加上 https://repo.huaweicloud.com/repository/maven/
原因是 阿里云 的镜像是伪镜像, 他会修改你的 maven pom, (比如把你的 latest 改成其他版本, 日常发癫)
而且 阿里云 的镜像中 某个库更新的时候, 更新完成前. 整个库的所有版本都会 404 (很傻逼)

**Comment by cssxsh** [comment]
https://docs.miraimc.dreamvoid.me/docs/configure-file.html

![image](https://user-images.githubusercontent.com/32539286/217190477-6895b533-6821-4e59-9ce7-733ba974b3f9.png)

可以自行调整 mirai 版本

另外 maven-repo-url 建议修改为 https://repo.huaweicloud.com/repository/maven/
